src/main/java/ortus/boxlang/runtime/config/segments/DatasourceConfig.java

```diff
@@ -542,15 +560,12 @@ public HikariConfig toHikariConfig() {
 			    "Datasource configuration must contain a 'driver', or a valid JDBC connection string in 'url'." );
 		}
 
-		DatasourceService	datasourceService	= BoxRuntime.getInstance().getDataSourceService();
-		HikariConfig		result				= new HikariConfig();
+		HikariConfig	result		= new HikariConfig();
 		// If we can't find the driver, we default to the generic driver
-		IJDBCDriver			driverOrDefault		= datasourceService.hasDriver( getDriver() )
-		    ? datasourceService.getDriver( getDriver() )
-		    : datasourceService.getGenericDriver();
+		IJDBCDriver		JDBCdriver	= getDriver();
```

Comment:
Ok, I see - this is effectively the same behavior, just moving the driver retrieval to a clean `getDriver()` method.

So we should still be able to initialize `DatasourceConfig` objects just fine. We won't be able to call `toHikariConfig()` until the JDBC module is loaded, but that was the case before and is completely understandable.